.github/workflows/e2e-nvidia-l40s-x8.yml

```diff
@@ -0,0 +1,218 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: E2E (NVIDIA L40S x8)
+
+on:
+  schedule:
+    - cron: '0 11 * * *' # Runs at 11AM UTC every day
```

Comment:
This might be better to kick off really early in the morning.  Maybe 1am?  Instance availability might be a little better and it should be finished during/by ET daylight hours.

Reply:
Updated to 6am UTC (aka 1am ET)!